test/e2e/pod_stats_test.go

```diff
@@ -16,10 +16,6 @@ var _ = Describe("Podman pod stats", func() {
 	)
 
 	BeforeEach(func() {
-		if os.Geteuid() != 0 {
```

Comment:
This test does not work rootless. Can we do a skip rootless here?

The underlying issue is that we're always using `--cgroup-manager=cgroupfs` when executing Podman and that doesn't work as an ordinary user.

Reply:
Re-added, although it would be good to figure out how we can get these tests to run with rootless and cgroups v2.

Reply:
Thanks!

Absolutely agree on getting to run in rootless mode.